### PR TITLE
Refactor for pylint

### PR DESCRIPTION
**Pull request by kequach**
https://github.com/UtrechtUniversity/SWORDS-UU/issues/36
This is still a draft PR but I'm opening this for review to let the pipeline run the linter

### CODE REVIEW

**Comment by kequach** [comment]
How to deal with the pyling error: "Catching too general exception Exception" ?
This is for example an issue when fetching data from GitHub:
image

Since there might be different kind of exceptions, I believe it is correct to catch as generic as possible. What do you think?

**Comment by kequach** [comment]
I'm also wondering whether we should disable this one:
image
I don't see a reason why such a rule should exist. I adjusted the code for a few of these but I find it completely unnecessary since it just bloats the code.

**Comment by kequach** [comment]
I also changed my linter to pylint instead of PEP8, to at least take care of some linting errors automatically.

**Comment by kequach** [comment]
While looking at this error: 
image

I was googling a bit and this answer I found speaks **against** using the suggestion from the linter: https://stackoverflow.com/a/11990189/5708610
Disable?

**Comment by kequach** [comment]
I'm done with most of it now, you can check the latest runs. Just these few issues mentioned here.

**Comment by J535D165** [comment]
Thanks a lot, this PR is a great step towards higher quality code. 

I think we can ignore some of the exceptions. 
The `no-else-return` is an interesting one, but, if possible, let's resolve.

**Comment by J535D165** [comment]
Let's ignore the `uu_api_crawler` for now. (Can we disable this file only?)

**Comment by kequach** [comment]
> While looking at this error: image
> 
> I was googling a bit and this answer I found speaks **against** using the suggestion from the linter: https://stackoverflow.com/a/11990189/5708610 Disable?

I also found this on a second google search: https://github.com/wemake-services/wemake-python-styleguide/issues/689
TLDR: enumerate is more pythonic. I will change it to enumerate.

**Comment by kequach** [comment]
@J535D165 we have a pass now! Ready to merge
